### PR TITLE
Remove NetworkBridge

### PR DESCRIPTION
**Pull request by NullSoldier**
The idea behind the network bridge, is that we'd move the Node entirely
inside of a worker, and then serialize messages across from the network,
to the worker, using the NetworkBridge. We've changed strategies and
are moving toward a WorkerPool approach, where we spin up workers
based on the number of cores, and move tasks that will freeze up the main
thread onto those workers.

### CODE REVIEW

File: ironfish/src/sdk.ts [hunk 1]
```diff
@@ -227,7 +238,7 @@ export class IronfishSdk {
 }
 
 /** Get the current platform or null if it cannot detect the platform */
```
Comment (dguenther) [inline ironfish/src/sdk.ts:240]: This comment is wrong now

File: ironfish/src/mining/director.ts [hunk 2]
```diff
@@ -147,6 +150,10 @@ export class MiningDirector<
         this.logger.error(err)
       })
     })
+
+    if (options.graffiti) {
+      this.setBlockGraffiti(options.graffiti)
+    }
```
Comment (dguenther) [inline ironfish/src/mining/director.ts:156]: should we do this with minerAccount as well?

**Comment by dguenther** [comment]
This includes several unrelated refactors, it'd be nice in the future if we could split those out into separate PRs.

File: ironfish/src/sdk.ts [hunk 3]
```diff
@@ -157,7 +162,11 @@ export class IronfishSdk {
   }
 
   async node({ databaseName }: { databaseName?: string } = {}): Promise<IronfishNode> {
+    const webSocket = (await require('ws')) as IsomorphicWebSocketConstructor
+    const webRTC = (await require('wrtc')) as IsomorphicWebRtc | undefined
```
Comment (dguenther) [inline ironfish/src/sdk.ts:166]: This variable casing doesn't match that of `WebRtcConnection` and `IsomorphicWebRtc`

**Review by dguenther — APPROVED** [review]

File: ironfish/src/mining/director.ts (hunk 2 above)
Comment (NullSoldier) [reply ironfish/src/mining/director.ts:156]: lets do it... I actually just did that in the PR that removes captain from the mining director

File: ironfish/src/sdk.ts (hunk 1 above)
Comment (NullSoldier) [reply ironfish/src/sdk.ts:240]: fixed

File: ironfish/src/mining/director.ts (hunk 2 above)
Comment (NullSoldier) [reply ironfish/src/mining/director.ts:156]: done

File: ironfish/src/sdk.ts (hunk 3 above)
Comment (NullSoldier) [reply ironfish/src/sdk.ts]: fixed

File: ironfish/src/mining/director.ts (hunk 2 above)
Comment (NullSoldier) [reply ironfish/src/mining/director.ts:157]: fixed

**Pull request merged**